platform/darwin/src/MGLGeoJSONSource.h

```diff
@@ -7,48 +7,53 @@ NS_ASSUME_NONNULL_BEGIN
 @protocol MGLFeature;
 
 /**
+ Options for `MGLGeoJSONSource` objects.
+ */
+typedef NSString *MGLGeoJSONSourceOption NS_STRING_ENUM;
+
+/**
  An `NSNumber` object containing a Boolean enabling or disabling clustering.
  If the `features` property contains point features, setting this option to
  `YES` clusters the points by radius into groups. The default value is `NO`.
  */
-extern NSString * const MGLGeoJSONClusterOption;
+extern MGLGeoJSONSourceOption const MGLGeoJSONClusterOption;
```

Comment:
f2bfc2c95d76a1066552220d4658cebb111bca97 renames these constants. Since they’re new to v3.4.0, I didn’t bother to keep the old names around.